### PR TITLE
Ingestion icon and font size changes

### PR DESCRIPTION
**Pull request by itsanudeep**
- Added new icon for ingestion tile
- Added an entry in menu for ingestion
- Adjusted font size in table view
- removed unused dependencies.

### CODE REVIEW

File: cdap-ui/app/cdap/components/AppHeader/AppDrawer/AppDrawer.tsx [hunk 1]
```diff
@@ -139,6 +140,17 @@ class AppDrawer extends React.PureComponent<IAppDrawerProps> {
...
           />
+          <ExperimentalFeature name={'data-ingestion'}>
+            <DrawerFeatureLink
+              featureName="Ingest"
+              featureUrl={`/${nsurl}/ingestion`}
+              featureSVGIconName="icon-ingest"
+              componentDidNavigate={componentDidNavigate}
+              data-cy="navbar-ingestion-link"
+              id="navbar-ingestion"
+              isActive={location.pathname === `/cdap/${nsurl}/ingestion`}
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/AppHeader/AppDrawer/AppDrawer.tsx]: i don't think this is needed if there's only one path

File: cdap-ui/app/cdap/components/Ingestion/PluginsTableView/index.tsx [hunk 2]
```diff
@@ -145,8 +148,12 @@ const PluginListView: React.FC<ICodeEditorProps> = ({
               });
 
               return (
-                <TableRow key={`${i}-${displayName}`} style={{ width: '100%' }}>
-                  <TableCell>
+                <TableRow
+                  key={`${i}-${displayName}`}
+                  style={{ width: '100%' }}
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/Ingestion/PluginsTableView/index.tsx]: can this width be set on the class?

File: cdap-ui/app/cdap/components/AppHeader/AppDrawer/AppDrawer.tsx (hunk 1 above)
Comment (itsanudeep) [reply cdap-ui/app/cdap/components/AppHeader/AppDrawer/AppDrawer.tsx]: done.

File: cdap-ui/app/cdap/components/Ingestion/PluginsTableView/index.tsx (hunk 2 above)
Comment (itsanudeep) [reply cdap-ui/app/cdap/components/Ingestion/PluginsTableView/index.tsx]: done.

**Review by elfenheart — APPROVED** [review]
👍 LGTM

**Comment by itsanudeep** [comment]
Build and Test: https://builds.cask.co/browse/CDAP-URUT237-1

**Pull request merged**